<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_dialog</class>
 <widget class="QDialog" name="main_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="wind_energy">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 200pt;</string>
   </property>
   <property name="text">
    <string>Wind Energy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="solar_energy">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 200pt;</string>
   </property>
   <property name="text">
    <string>Solar Energy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hydro_energy">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 200pt;</string>
   </property>
   <property name="text">
    <string>Hydro Energy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="biomass_energy">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 200pt;</string>
   </property>
   <property name="text">
    <string>Biomass Energy </string>
   </property>
  </widget>
  <widget class="QPushButton" name="geothermal_energy">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 200pt;</string>
   </property>
   <property name="text">
    <string>Geothermal Energy</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="top_10_area">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>460</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>139</number>
   </attribute>
   <column>
    <property name="text">
     <string>Country</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Year</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Growth(%)</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="least_10_area">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>190</y>
     <width>460</width>
     <height>331</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>139</number>
   </attribute>
   <column>
    <property name="text">
     <string>Country</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Year</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Growth(%)</string>
    </property>
   </column>
  </widget>
  <widget class="QPlainTextEdit" name="project_description">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:12pt;
</string>
   </property>
   <property name="plainText">
    <string>This is a data analysis interface with five distinct datasets.</string>
   </property>
  </widget>
  <widget class="QLabel" name="top_10">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Top 10 Countries by Growth</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="least_10">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>130</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Least 10 Countries by Growth</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="project_description_title">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Project Description</string>
   </property>
  </widget>
  <widget class="QPushButton" name="top_10_graph">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>530</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 200pt;</string>
   </property>
   <property name="text">
    <string>Graph</string>
   </property>
  </widget>
  <widget class="QPushButton" name="least_10_graph">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>530</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 200pt;</string>
   </property>
   <property name="text">
    <string>Graph</string>
   </property>
  </widget>
  <widget class="QLabel" name="windenergy_icon">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Icons/1wind-energy.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="solar_energy_icon">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Icons/2solar-energy.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="hydro_energy_icon">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Icons/3hydro-power.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="biomass_energy_icon">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Icons/4biomass.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="geothermal_energy_icon">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Icons/5geothermal-energy.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="background_image">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1191</width>
     <height>651</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1191</width>
     <height>651</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1191</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Icons/pexels-flickr-157039.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="project_title">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>651</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Renewables Energy - Data Analysis Tool</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>background_image</zorder>
  <zorder>wind_energy</zorder>
  <zorder>solar_energy</zorder>
  <zorder>hydro_energy</zorder>
  <zorder>biomass_energy</zorder>
  <zorder>geothermal_energy</zorder>
  <zorder>top_10_area</zorder>
  <zorder>least_10_area</zorder>
  <zorder>project_description</zorder>
  <zorder>top_10</zorder>
  <zorder>least_10</zorder>
  <zorder>project_description_title</zorder>
  <zorder>top_10_graph</zorder>
  <zorder>least_10_graph</zorder>
  <zorder>windenergy_icon</zorder>
  <zorder>solar_energy_icon</zorder>
  <zorder>hydro_energy_icon</zorder>
  <zorder>biomass_energy_icon</zorder>
  <zorder>geothermal_energy_icon</zorder>
  <zorder>project_title</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
